### DISCUSSION TITLE
Generating sealed secret for password which contains special character('&,'!')

### DISCUSSION THREAD

**Discussion by sankarraman87** [bitnami-labs/sealed-secrets#1276] [Q&A]
We are using below command to create sealed secret for password which contains special character(&). Created encrypted value is not working when we use it on Springboot application. It is working if we hardcoded password value. I tried to use \& to escape & character but still it is not working. Could you please help me how to create encrypted value for password which contains special character ?

echo -n 087Icy&rX5qG | kubeseal --raw --scope --cluster-wide

Output:
[1] 34476
087IcY-bash: rX5qG: command not found
AgCI4PP/zF25aE/BsUONuu***************************************[1]+  Done

**Comment by theurichde** [comment]
Try quoting your input.

`echo -n '087Icy&rX5qG' | kubeseal --raw --scope cluster-wide`
